.github/workflows/test.yml

```diff
@@ -6,7 +6,7 @@ name: CI
 on:
   # Triggers the workflow on push or pull request events but only for the master branch
   push:
-    branches: [ master ]
+    branches: "*"
```

Comment:
why is this changed?

Reply:
This was just for testing on my repo, but should be deleted upon merge.